index.js

```diff
@@ -10,12 +10,14 @@ window.Stimulus = application;
 // Import all controller files
 import AccordionController from "./lib/rbui/accordion/accordion_controller";
 import AlertDialogController from "./lib/rbui/alert_dialog/alert_dialog_controller";
+import CalendarController from "./lib/rbui/calendar/calendar_controller";
 import CollapsibleController from "./lib/rbui/collapsible/collapsible_controller";
 import CheckboxGroupController from "./lib/rbui/checkbox/checkbox_group_controller";
 import ComboboxController from "./lib/rbui/combobox/combobox_controller";
 import ComboboxContentController from "./lib/rbui/combobox/combobox_content_controller";
 import ComboboxItemController from "./lib/rbui/combobox/combobox_item_controller";
 import FormFieldController from "./lib/rbui/form/form_field_controller";
+import InputController from "./lib/rbui/input/input_controller";
```

Comment:
@jacksonpires , could you rename this controller to "calendar_input_controller"? Input is a generic name and we don't like to use it outside of calendar